documentation/ame-guide/modules/ROOT/pages/modeling/connect-elements.adoc

```diff
@@ -0,0 +1,40 @@
+= Connect elements
+
+The elements of an Aspect Model &ndash; such as Properties, Operations, Characteristics, Constraints, Entities, or Events &ndash;
+can be distributed over the complete editor main canvas.
+
+To connect two elements manually, proceed as follows:
+
+. Click both elements while holding the CTRL key on your keyboard.
+. Click the *Connect* icon in the xref:getting-started/ui-overview.adoc#toolbar[toolbar].
++
+image::connecting-elements.png[width=100%]
+
+→ Both elements are now connected.
+
+* If only one direction is permitted, the arrow will be set respectively.
+* If both directions are permitted, the arrow will be set for the element selected first to the element selected second.
+* Tf a connection is not permitted, a respective warning will appear and the elements will not be connected.
+
+TIP: Alternatively, you can use the context menu (right mouse click) to access the action items.
+
+The larger the diagram, the harder it might get elements that are fare away.
+
+In case you cannot connect two elements just the way you are most familiar with, try one of these procedures.
+
+== Connect via context menu
+
+* Click the source element.
+* Right-click to access the context menu.
+* Select *connect with ...*.
+* The new pop-up displays all the elements of the model. +
+  In case there are too element many type the name in the search field.
```

Comment:
This sentence sounds strange. "In case there are too many elements, type..."?

Reply:
Will be added later.
Not part of this story.